spec/support/api/schemas/artists.json

```diff
@@ -1,42 +1,89 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "id": "http://spotify-apprentice-app.com",
+  "id": "http://jsonschema.net",
   "type": "object",
   "properties": {
+    "songs": {
+      "id": "http://jsonschema.net/songs",
+      "type": "array",
+      "items": [
+        {
+          "id": "http://jsonschema.net/songs/0",
+          "type": "object",
+          "properties": {
+            "id": {
+              "id": "http://jsonschema.net/songs/0/id",
+              "type": "string"
+            },
+            "name": {
+              "id": "http://jsonschema.net/songs/0/name",
+              "type": "string"
+            },
+            "spotify_uri": {
+              "id": "http://jsonschema.net/songs/0/spotify_uri",
+              "type": "string"
+            },
+            "artist_id": {
+              "id": "http://jsonschema.net/songs/0/artist_id",
+              "type": "string"
+            }
+          },
+          "required": [
+            "id",
+            "name",
+            "spotify_uri",
+            "artist_id"
+          ]
+        }
+      ]
+    },
     "artists": {
-      "id": "http://spotify-apprentice-app.com/artists",
+      "id": "http://jsonschema.net/artists",
       "type": "array",
       "items": {
-        "id": "http://spotify-apprentice-app.com/artists/0",
+        "id": "http://jsonschema.net/artists/0",
         "type": "object",
         "properties": {
           "id": {
-            "id": "http://spotify-apprentice-app.com/artists/0/id",
+            "id": "http://jsonschema.net/artists/0/id",
             "type": "string"
           },
           "name": {
-            "id": "http://spotify-apprentice-app.com/artists/0/name",
+            "id": "http://jsonschema.net/artists/0/name",
             "type": "string"
           },
           "description": {
-            "id": "http://spotify-apprentice-app.com/artists/0/description",
+            "id": "http://jsonschema.net/artists/0/description",
             "type": "string"
           },
           "spotify_uri": {
-            "id": "http://spotify-apprentice-app.com/artists/0/spotify_uri",
+            "id": "http://jsonschema.net/artists/0/spotify_uri",
             "type": "string"
+          },
+          "song_ids": {
+            "id": "http://jsonschema.net/artists/0/song_ids",
+            "type": "array",
+            "items": [
+              {
+                "id": "http://jsonschema.net/artists/0/song_ids/0",
+                "type": "string"
+              },
+              {
+                "id": "http://jsonschema.net/artists/0/song_ids/1",
```

Comment:
If you paste in JSON with an array containing 3 objects, the schema generator will assume that there should be 3 items in the JSON. It is annoying.  I'd delete lines 71-78 as they're duplicative.